src/reducers/app.js

```diff
@@ -0,0 +1,62 @@
+import { Record, List } from 'immutable';
+import enUS from 'antd/lib/locale-provider/en_US';
+import frFR from 'antd/lib/locale-provider/fr_FR';
+import getTranslations, { getAvailableLocale } from '../utils/locales';
+
+const SET_LOCALE = 'faucet/locale/set';
+
+export const App = new Record({
```

Comment:
i think it makes more sense to use a Map in both this case as well as with User. Record adds complexity & a performance penalty.

Reply:
Ok, hopped off the bandwagon. Maybe save it for something juicy later.
https://tonyhb.gitbooks.io/redux-without-profanity/content/using_immutablejs_records.html